cmd/compute-domain-kubelet-plugin/checkpoint.go

```diff
@@ -12,14 +13,22 @@ type Checkpoint struct {
 }
 
 type CheckpointV1 struct {
-	PreparedClaims PreparedClaims `json:"preparedClaims,omitempty"`
+	PreparedClaimsByUID PreparedClaimsByUID `json:"preparedClaimsByUID,omitempty"`
```

Comment:
Can we drop the `ByUID` in the field name (but keep it in the type)? If you feel strongly we can keep it, but it feels a bit awkward to have this detail exposed in the actual checkpointed file (where its contents will clearly be indexing here by uuid).